### PR TITLE
Update readme with everything that is installed

### PR DESCRIPTION
**Pull request by carmee-en**
Related to #1 

This Pull Request adds more info about which apps are being installed using Homebrew. Also, it adds info on the dotfiles it installs exactly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,24 @@ curl -sL https://github.com/PrivacyCloud/.dotfiles/archive/master.tar.gz | tar x
...
+      - Zsh
+      - Yarn
+      - Docker
+      - Google Chrome
+      - Java
+      - Postman
+      - Slack
+      - Visual Studio Code
+   - [nvm](https://github.com/creationix/nvm)
+   - [zsh]() and [oh my zsh](https://github.com/ohmyzsh/ohmyzsh)
```
Comment (SantiMA10) [inline README.md]: We miss the zsh link, isn't it?

**Review by SantiMA10 — CHANGES_REQUESTED** [review]
One comment and ready 🚀

File: README.md (hunk 1 above)
Comment (carmee-en) [reply README.md]: Done! I also added a link for `yarn`

**Review by dcarral — APPROVED** [review]
Awesome @carmee-en, keep up the good work! :)

**Comment by dcarral** [comment]
By the way @carmee-en: useful tip & trick to automatically close issues from PRs @ GitHub: https://stackoverflow.com/questions/12235620/automatically-closing-issue-from-pull-request-in-github

**Pull request merged**